packages/js/components/src/split-dropdown/README.md

```diff
@@ -0,0 +1,29 @@
+SplitDropdown
+===
+
+This is a ButtonGroup with a primary action and a menu hidden behind a Dropdown toggle button. The component expects to receive elements (probably Buttons) as children. The primary action will be the first child, and the dropdown menu will contain the rest of the children.
+
+## Usage
+
+```jsx
+    // Primary button variant (default)
+    <SplitDropdown>
+        <Button onClick={ primaryActionHandler } text="Primary Action"></Button>
```

Comment:
This way it would act more as a Button component with a dropdown attached.

Reply:
Letting folks pass in a fully-qualified React element for the primary action means people can use Button properties that we haven't specifically prepared for, or components we haven't considered that duck-type as Buttons.

Is it a little weird that we're taking the first child and doing something with it that's different from what we do with the others? Maybe. Obviously it didn't bother me as I was writing it, but maybe it depends on how you think about the actions. Is the primary action the main thing you care about, with all others mere afterthoughts, or is it just the first among equals? Is the SplitDropdown a button that does button things but also has an Advanced mode, or is it a ButtonGroup condensed down into a smaller space?

I don't have good definitive answers for any of this, of course. I will say that asking the dev for props that we then use to create a component for them--a component of a type they're probably already using elsewhere on the page--_feels_ like working against React rather than with it.

Reply:
> Letting folks pass in a fully-qualified React element for the primary action means people can use Button properties that we haven't specifically prepared for

This is formal acknowledgement that something like `primaryActionProps` would also accomplish this goal.